f5_openstack_agent/lbaasv2/drivers/bigip/network_helper.py

```diff
@@ -357,17 +357,15 @@ def arp_delete(self,
                    ip_address,
                    partition=const.DEFAULT_PARTITION):
         if ip_address:
-            address = urllib.quote(self._remove_route_domain_zero(ip_address))
-            arp = bigip.tm.net.arps.arp
             try:
-                if arp.exists(name=address, partition=partition):
-
-                    arp.load(name=address, partition=partition)
-                    arp.delete()
+                arps = bigip.tm.net.arps.get_collection(partition=partition)
+                for arp in arps:
+                    if arp.ipAddress == ip_address:
+                        arp.delete()
```

Comment:
The GUI allows me to create a static arp entry with '%0' for the default domain but strips it out automatically.  It keeps anything else, like '%1'.

It looks like arp_delete might be called with an ip_address always containing '%<num', even for default domain.  That means the for loop won't delete arp entries for the common partition such as for shared network across tenants.